cedar-policy-validator/src/schema_file_format.rs

```diff
@@ -490,26 +490,6 @@ impl<N> SchemaType<N> {
         }
     }
 
-    /// Is this [`SchemaType`] an extension type, or does it contain one
```

Comment:
Looks like the downstream DRT failure is because it needs this function. Keep it unless you've got something else in mind?

Reply:
Just thought this was unused, so removed it. Didn't think to check whether DRT used it.